migrations/create_bouncer_tables.php

```diff
@@ -56,7 +56,7 @@ public function up()
         });
 
         Schema::create(Models::table('permissions'), function (Blueprint $table) {
-            $table->integer('ability_id')->unsigned()->index();
+            $table->unsignedInteger('ability_id')->index();
             $table->unsignedInteger('entity_id');
```

Comment:
@JosephSilber As we use once, why not use and standardize? :smile:

Reply:
I find single-word method to be more legible.

Reply:
If you want to to you can change the PR the other way around: change all `unsignedInteger`s to `integer(...)->unsigned()`.